app/commands/v2/publish.rb

```diff
@@ -101,6 +102,18 @@ def build_live_attributes(draft_content_item)
 
         attributes
       end
+
+      def handle_path_change(live_content_item)
+        path_change = live_content_item.previous_changes[:base_path]
+        if path_change.present? && path_change[0].present?
+          draft_redirect = DraftContentItem.find_by(format: "redirect", base_path: path_change[0])
+          self.class.call(
+            content_id: draft_redirect.content_id,
+            locale: draft_redirect.locale,
+            update_type: "major"
```

Comment:
Why does this happen as a "major" change? Shouldn't it use whatever `update_type` was used by the client?